crates/chain/src/indexer.rs

```diff
@@ -0,0 +1,31 @@
+use bitcoin::{OutPoint, Transaction, TxOut};
```

Comment:
❓ Why not make this file `src/indexer/mod.rs` ?

Reply:
Done in 884ac9009350d2e5fb0b7459935905216d5fcf2e